### PR TITLE
Add zfit

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/zfit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  license_family: BSD
+  license_file: LICENSE
+  summary: Physics extension to zfit
+  doc_url: https://zfit.readthedocs.io/
+  dev_url: https://github.com/zfit/zfit
+
+extra:
+  recipe-maintainers:
+    - chrisburr
+    - mayou36
```
Comment (chrisburr) [inline recipes/zfit/meta.yaml]: @mayou36 Can you confirm here that you're happy to be listed as a maintainer?

Comment (mayou36) [reply recipes/zfit/meta.yaml]: I confirm ;)

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes, this is ready for review

**Comment by chrisburr** [comment]
Thank you @beckermr! 🍰

**Pull request merged**